states/MI/governor.yaml

```diff
@@ -0,0 +1,139 @@
+contact_form:
+  steps:
+    - visit: "https://somgovweb.state.mi.us/GovRelations/ShareOpinion.aspx"
+    - fill_in:
+        - name: "ctl00$Content$OpinionPrefix"
+          selector: "#Content_OpinionPrefix"
+          value: $NAME_PREFIX
+          required: true
+        - name: "ctl00$Content$OpinionFirstName"
+          selector: "#Content_OpinionFirstName"
+          value: $NAME_FIRST
+          required: true
+        - name: "ctl00$Content$OpinionLastName"
+          selector: "#Content_OpinionLastName"
+          value: $NAME_LAST
+          required: true
+        - name: "ctl00$Content$OpinionAddress"
+          selector: "#Content_OpinionAddress"
+          value: $ADDRESS_STREET
+          required: true
+        - name: "ctl00$Content$OpinionCity"
+          selector: "#Content_OpinionCity"
+          value: $ADDRESS_CITY
+          required: true
+    - select:
+        - name: "ctl00$Content$OpinionState"
+          selector: "#Content_OpinionState"
+          value: $ADDRESS_STATE_FULL
```

Comment:
I think this should be technically be $ADDRESS_STATE_POSTAL_ABBREV?

Reply:
Well, the way Mercury works, it can be either : )
I think using the abbreviation is better because e.g. DC can have a few different spelled-out names and the abbreviation is more unambiguous.

Reply:
Should be the abbreviation, you're right.  I'll change it, even if mercury can figure it out, it should be inline with the page selection.